.circleci/config.yml

```diff
@@ -4,6 +4,22 @@ orbs:
   secrethub: secrethub/cli@1.1.0
   gh: circleci/github-cli@2.0.0
 
+commands:
+  yarn-install:
+    description: Runs Yarn install
+    steps:
+      - restore_cache:
+          name: Restore Yarn cache
+          keys:
+            - gio-ui-components-yarn-cache-v1-{{ checksum "yarn.lock" }}
+      - run:
+          name: 'Install Yarn dependencies'
+          command: yarn install
+      - save_cache:
+          paths:
+            - .yarn/cache
+          key: gio-ui-components-yarn-cache-v1-{{ checksum "yarn.lock" }}
```

Comment:
Not sure we need a command as we are only using it once 🤷🏻

Reply:
I made myself the same reflexion but IMHO that it is better if it is the same everywhere. in case of improvement or other .. 🤷‍♂️